templates/components/account/order-contents.html

```diff
@@ -21,14 +21,18 @@ <h5 class="account-listShipping-title">{{lang 'account.orders.details.ship_to_mu
             </div>
             <figure class="account-product-figure">
                 {{#if type '===' 'giftcertificate'}}
-                    <img src="{{cdn ../../theme_settings.default_image_gift_certificate}}" alt="Gift Certificate" title="Gift Certificate">
+                    <img
+                        src="{{cdn ../theme_settings.default_image_gift_certificate}}"
+                        alt="Gift Certificate"
+                        title="Gift Certificate"
+                    >
                 {{else}}
                     {{> components/common/responsive-img
                         image=image
                         class="account-product-image"
-                        fallback_size=../../theme_settings.productthumb_size
-                        lazyload=../../theme_settings.lazyload_mode
-                        default_image=../../theme_settings.default_image_product
+                        fallback_size=../theme_settings.productthumb_size
+                        lazyload=../theme_settings.lazyload_mode
+                        default_image=../theme_settings.default_image_product
```

Comment:
Does `@root` work in v4? It might make for cleaner syntax if we just did `@root.theme_settings.default_image_product` for example.

https://handlebarsjs.com/api-reference/data-variables.html

Reply:
Didn't check, will take a look.

Reply:
Just checked this, for some reason @root is undefined for v3 and v4.